### ISSUE TITLE
Get a11y working

### ISSUE THREAD

**Issue by timblack1** [timblack1/rcl#25]
Currently the accessibility element either works for the whole app, including the search text box, or it doesn't work at all.  We need to target it to a container element which does not contain the text box.
